.gitmodules

```diff
@@ -62,3 +62,6 @@
 [submodule "cmake/external/onnx-tensorrt"]
 	path = cmake/external/onnx-tensorrt
 	url = https://github.com/stevenlix/onnx-tensorrt.git
+[submodule "cmake/external/libprotobuf-mutator"]
```

Comment:
Can you add this new dependency in the cgmanifest.json file?

Reply:
I was planning to help him to that. He may be not familar with the process.